RockLib.Messaging/ForwardingReceiverMessage.cs

```diff
@@ -164,26 +166,24 @@ public async Task RejectAsync(CancellationToken cancellationToken)
 
         private Task HandleForwardedMessageAsync(ForwardingOutcome outcome, CancellationToken cancellationToken)
         {
-            switch (outcome)
+            return outcome switch
```

Comment:
Cool.  I didn't know C# had this type of switch as well.

Reply:
Agreed, this is neat!

Reply:
Pattern matching keeps getting better in C#. And fortunately, VS has a refactoring to do this automatically (that's what I used to change this :) ).